RIOT-OS/RIOT
# dist/tools/{ethos,zep_dispatch}: allow setting host ip

### Contribution description

This adds an optional cmd option to `start_network.sh` for both `ethos` and `zep_dispatch` which allows to automatically set an IP address for the host device (usually the development PC) when creating the tap interface.

`examples/gnrc_border_router` is adapted to showcase the usage.


### Testing procedure

1. `make -C examples/gnrc_border_router all term` to see it is still working without setting a host ip address.
2. `make -C examples/gnrc_border_router IPV6_HOST_ADDR=2001:db8:1::1 all term`, then `ip a l tap0` in a separate shell to see the host ip address. `ping 2001:db8:1::1` in the RIOT shell to see it is reachable.
3. same commands with an actual board (tested with `BOARD=nrf52840dk`) to test `ethos`.


### Issues/PRs references

This has been discussed with @miri64 previously.
